src/main/resources/templates/islieb/image.html

```diff
@@ -0,0 +1,3 @@
+<div style="display:flex;justify-content:center;height:100%;" xmlns:th="http://www.w3.org/1999/xhtml">
```

Comment:
replace with

```
<div xmlns:th="http://www.thymeleaf.org" style="display:flex;justify-content:center;height:100%;">
  <img th:src="${image.src}" src="https://islieb.de/blog/wp-content/uploads/2018/03/isfies-familienurlaub.png"
       style="max-height:100%;width:auto;"/>
</div>
```

correct xml:th and a default src that will be overriden by th:src